src/eventEmitter/index.js

```diff
@@ -129,7 +129,7 @@ if (typeof window === 'undefined') {
     listeners = this._events && this._events[eventName];
 
     if (!listeners || !listeners.length) {
-      return;
+      return false;
```

Comment:
fix KuzzleEventEmitter methods signatures, to be consistant with NodeJS `event` specitications:

emit should return a boolean value.
(see https://nodejs.org/api/events.html#events_emitter_emit_eventname_args )